x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryFolderTests.java

```diff
@@ -430,4 +430,17 @@ public void testFoldingOfPivot() {
         assertThat(a, containsString("\"terms\":{\"field\":\"keyword\""));
         assertThat(a, containsString("{\"avg\":{\"field\":\"int\"}"));
     }
+
+    public void testFoldingOfLiteralWithGroupBy() {
```

Comment:
You can remove this, as it's already added here: https://github.com/elastic/elasticsearch/blob/26283b63882e85ec7aca09cae8bd046f01bdd8c9/x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryFolderTests.java#L383